src/index.html

```diff
@@ -1,12 +1,363 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="main-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="main-page__content">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="is-active nav__link"  href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+      <main>
+        <section class="container">
```

Comment:
will be better to use more clear and understandable names for bem blocks, for example, `catalog`